deploy/k8s/bus-bench/main.go

```
@@ -0,0 +1,519 @@
+}
+
+func runPublish(url, subject string, duration time.Duration, startAt int64, payloadSize, confirmEvery, rate, podIndex, feeders int) error {
+	waitUntil(startAt)
+	windowStart := time.Now()
+	deadline := windowStart.Add(duration)
+
+	pub, err := bus.NewPublisher(url, zap.NewNop())
+	if err != nil {
+		return err
+	}
+	defer pub.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `pub.Close` is not checked

(errcheck)